src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetRandomCommand.cs

```diff
@@ -275,7 +276,7 @@ private double ConvertToDouble(object o, double defaultIfNull)
         /// List from which random elements are chosen.
         /// </summary>
         [Parameter(ParameterSetName = RandomListItemParameterSet, ValueFromPipeline = true, Position = 0, Mandatory = true)]
-        [ValidateNotNullOrEmpty]
+        [System.Management.Automation.AllowNull]
```

Comment:
```suggestion
        [AllowNull]
```

Reply:
@iSazonov: There's a conflict with `System.Diagnostics.CodeAnalysis.AllowNullAttribute`, hence use of the full type name.

 👍 on the other suggestions.